File: src/main/java/itmo/is/project/mapper/module/production/ProductionModuleBlueprintMapper.java
package itmo.is.project.mapper.module.production;

import itmo.is.project.dto.module.production.ProductionModuleBlueprintDto;
import itmo.is.project.mapper.EntityMapper;
import itmo.is.project.model.module.production.ProductionModuleBlueprint;
import org.mapstruct.Mapper;

@Mapper
public interface ProductionModuleBlueprintMapper
        extends EntityMapper<ProductionModuleBlueprintDto, ProductionModuleBlueprint> {
}

File: src/main/kotlin/itmo/isproject/production/mapper/ProductionModuleBlueprintMapper.kt
package itmo.isproject.production.mapper

import itmo.isproject.production.dto.ProductionModuleBlueprintDto
import itmo.isproject.shared.common.mapper.EntityMapper
import itmo.isproject.production.model.ProductionModuleBlueprint
import org.mapstruct.Mapper

@Mapper
interface ProductionModuleBlueprintMapper : EntityMapper<ProductionModuleBlueprintDto, ProductionModuleBlueprint>
